src/managers/XCursorManager.cpp

```diff
@@ -542,3 +549,56 @@ std::vector<SP<SXCursors>> CXCursorManager::loadAllFromDir(std::string const& pa
 
     return newCursors;
 }
+
+void CXCursorManager::syncGsettings() {
+    auto checkParamExists = [](std::string const& paramName, std::string const& category) {
+        auto* gSettingsSchemaSource = g_settings_schema_source_get_default();
```

Comment:
do we need to unref this?

Reply:
https://docs.gtk.org/gio/type_func.SettingsSchemaSource.get_default.html?q=g_settings_schema_source_lookup

"The data is owned by the called function."

so nope.

Reply:
ok